lib/src/chart/pie_chart/pie_chart_data.dart

```diff
@@ -31,6 +31,10 @@ class PieChartData extends BaseChartData with EquatableMixin {
     PieTouchData? pieTouchData,
     FlBorderData? borderData,
     bool? titleSunbeamLayout,
+    CapStyle? capStyle,
+    bool? shadow,
```

Comment:
Isn't it better to use the [Shadow](https://api.flutter.dev/flutter/dart-ui/Shadow-class.html) object?

Something like what we have in the LineChart:
https://github.com/imaNNeo/fl_chart/blob/11d13a78e6a134aa0cfdcd8dfa88dceaf79a4033/lib/src/chart/line_chart/line_chart_data.dart#L360-L361

And this is how we paint it in the `drawBarShadow()` function:
https://github.com/imaNNeo/fl_chart/blob/11d13a78e6a134aa0cfdcd8dfa88dceaf79a4033/lib/src/chart/line_chart/line_chart_painter.dart#L906-L939

Reply:
And if it is transparent (default value), we will not draw it.